app/src/test/java/de/danoeh/antennapod/ui/cleaner/ShownotesCleanerTest.java

```diff
@@ -18,6 +19,7 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
+
 /**
```

Comment:
Please revert unrelated change

```suggestion

/**
```